src/Smuggler/Plugin.hs

```diff
@@ -121,3 +119,6 @@ unusedLocs (L (RealSrcSpan loc) decl, used, unused)
         L _ (IEName (L (RealSrcSpan lieLoc) name)) <- [lieName]
         guard $ name `elem` unused
         pure (srcSpanStartLine lieLoc, srcSpanStartCol lieLoc)
+    getEndColMax u = maximum $ map (findColLoc . nameSrcSpan) u
```

Comment:
I'm a bit worried about the usage of partial function `maximum` but maybe the usage is completely correct in practice

Reply:
I added custom function to calculate maximum list item or give a default value instead.